sheriff_test.go

```diff
@@ -165,6 +167,42 @@ func TestMarshal_GroupsNoGroups(t *testing.T) {
 		},
 		"omit_empty":            "OmitEmpty",
 		"omit_empty_group_test": "OmitEmptyGroupTest",
+		"include_empty_tag":     "IncludeEmptyTag",
+	})
+	assert.NoError(t, err)
+
+	assert.Equal(t, string(expected), string(actual))
+}
+
+func TestMarshal_GroupsNoGroupsIncludeEmptyTag(t *testing.T) {
+	testModel := &TestGroupsModel{
+		DefaultMarshal:     "DefaultMarshal",
+		OnlyGroupTest:      "OnlyGroupTest",
+		GroupTestAndOther:  "GroupTestAndOther",
+		OmitEmpty:          "OmitEmpty",
+		OmitEmptyGroupTest: "",
+		SliceString:        []string{"test", "bla"},
+		IncludeEmptyTag:    "IncludeEmptyTag",
+	}
+
+	o := &Options{
+		IncludeEmptyTag: true,
+		Groups:          []string{"test"},
+	}
+
+	actualMap, err := Marshal(o, testModel)
+	assert.NoError(t, err)
+
+	actual, err := json.Marshal(actualMap)
+	assert.NoError(t, err)
+
+	expected, err := json.Marshal(map[string]interface{}{
+		"default_marshal":      "DefaultMarshal",
+		"only_group_test":      "OnlyGroupTest",
+		"group_test_and_other": "GroupTestAndOther",
+		"omit_empty":           "OmitEmpty",
+		"slice_string":         []string{"test", "bla"},
+		"include_empty_tag":    "IncludeEmptyTag",
 	})
 	assert.NoError(t, err)
```

Comment:
I think you'd need to add the value for the `IncludeEmptyTag` field to that test for it to be tested, no?
Otherwise the field is empty and not tested in that way.

Reply:
ah, sorry. Missed the part about `DefaultMarshal`. Right.

Reply:
all good then, my bad.